### PR TITLE
[ADF-4559] Add chips to multivalue metadata properties

### PR DESCRIPTION
**Pull request by davidcanonieto**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-4559

**What is the new behaviour?**
Add the possibility of displaying multivalue metadata properties in chips. They are dynamic and so when the property is in edit mode these chips can also be deleted selecting directly the chip and clicking in the button by its right side.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4559

### CODE REVIEW

File: lib/core/card-view/components/card-view-textitem/card-view-textitem.component.html [hunk 1]
```diff
@@ -1,88 +1,131 @@
...
+<div [attr.data-automation-id]="'card-textitem-label-' + property.key"
+     class="adf-property-label"
+     *ngIf="showProperty() || isEditable()">{{ property.label | translate }}</div>
 <div class="adf-property-value">
     <span *ngIf="!isEditable()">
-        <span *ngIf="!isClickable(); else elseBlock" [attr.data-automation-id]="'card-textitem-value-' + property.key">
-            <span *ngIf="showProperty()"
-                [ngClass]="property.multiline?'adf-textitem-multiline':'adf-textitem-scroll'">
-                {{ property.displayValue }}</span>
+        <span *ngIf="!isClickable(); else elseBlock"
```
Comment (DenysVuika) [inline lib/core/card-view/components/card-view-textitem/card-view-textitem.component.html]: I know it's not part of your changes, but could you please make `elseBlock` more meaningful and rename to something like `unclickable` or similar?

File: lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts [hunk 2]
```diff
@@ -133,4 +157,8 @@ export class CardViewTextItemComponent implements OnChanges {
             this.cardViewUpdateService.clicked(this.property);
         }
     }
+
+    isChipViewEnabled(): boolean {
```
Comment (DenysVuika) [inline lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts]: maybe worth turning into the property?
```suggestion
    get isChipViewEnabled(): boolean {
```

File: lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts [hunk 3]
```diff
@@ -87,19 +91,27 @@ export class CardViewTextItemComponent extends BaseCardView<CardViewTextItemMode
...
+        } else {
+            this.editedValue = this.property.multiline ? this.property.displayValue : this.property.value;
+        }
+    }
+
     private resetErrorMessages() {
         this.errorMessages = [];
     }
 
-    update(event: MouseEvent|KeyboardEvent): void {
```
Comment (eromano) [inline lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts]: why this type is changed?

**Comment by davidcanonieto** [comment]
> Can you make the chips the default behaviour ?

Done, now the chips are the default behavior.

**Pull request merged**